Java/java_solutions_1-10/Euler_4.java

```diff
@@ -7,36 +7,28 @@
 
 public class Euler_4
 {
-    static int largestPalindrome = 0;
-
-    public static void findLargestPalindrome(int m1, int m2)
-    {
-        Integer threeDigitProduct = m1 * m2;
-        if (threeDigitProduct < largestPalindrome) return;
-
-        String productStr = threeDigitProduct.toString();
-
-        int productStrLen = productStr.length();
-        int j = productStrLen - 1;
-
-        for (int i = 0; i < productStrLen; i++) {
-            if (productStr.charAt(i) != productStr.charAt(j)) 
-                return;
-            j--;
-        }
-
-        largestPalindrome = threeDigitProduct;
+    public static boolean isPalindrome(String candidate) {
+        return new StringBuilder(candidate).reverse()
```

Comment:
[StringBuilder](https://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html) newly used and researched.